app/grandchallenge/algorithms/views.py

```diff
@@ -109,6 +184,45 @@ def form_valid(self, form):
         return response
 
 
+class PhaseAlgorithmCreate(
+    LoginRequiredMixin,
+    PhaseConfiguredForAlgorithmSubmissionMixin,
+    UserCanSubmitToPhaseMixin,
+    VerificationRequiredMixin,
```

Comment:
Have you double checked that using multiple `UserPassesTestMixin` works? I seem to remember there being issues with this a while back that only the first one would be applied.

Reply:
It might not be worth separating these out into mixins yet, if the functionality is not being used it is premature optimisation.

Reply:
Yes, this is what I was alluding to with my comment above. It works, but it requires explicitly calling `super().test_func()` and then checking the the returned response. If you don't that, indeed only the last mixin that is called will be applied. I agree that it might not be worth seperating at the moment, so I'll combine them into one (also the verification check).